build/build.proj

```diff
@@ -311,6 +335,27 @@
     </MSBuild>
   </Target>
 
+  <!--
+    ============================================================
+    Restore package signing cross framework verify test project
+    ============================================================
+  -->
+  <Target Name="RestoreCrossVerifyTest">
```

Comment:
Are we using this target anywhere in code after defining?

Reply:
Good catch! Thanks! Fixed.